<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FamilyCreationForm</class>
 <widget class="QDialog" name="FamilyCreationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>188</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>188</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>188</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Creation Famille d'Article</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Reference</string>
   </property>
   <property name="buddy">
    <cstring>ledit_reference</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledit_reference">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>15</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Libele</string>
   </property>
   <property name="buddy">
    <cstring>ledit_designation</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledit_designation">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>561</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>65</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
   <property name="buddy">
    <cstring>description</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_quit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Quiter</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/product/cancel.png</normaloff>:/product/cancel.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_clear">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Effacer</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/product/clear.png</normaloff>:/product/clear.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_save">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/product/validate.png</normaloff>:/product/validate.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
